raiden/ui/cli.py

```diff
@@ -173,7 +173,7 @@ def wait_for_sync_etherscan(
 ) -> None:
     local_block = blockchain_service.client.block_number()
     etherscan_block = etherscan_query_with_retries(url, sleep)
-    syncing_str = 'Syncing ... Current: {} / Target: ~{}'
+    syncing_str = '\rSyncing ... Current: {} / Target: ~{}'
```

Comment:
If you want to use carriage return, could you please remove the line which is printing the ansi codes?

Reply:
done